db/config.json

```diff
@@ -3,7 +3,8 @@
     "dialect": "postgres",
     "use_env_variable": "DATABASE_URL",
 		"pool": {
-			"max": 1,
+			"acquire": 60000,
+			"max": 15,
```

Comment:
For my benefit lolol, why 60000 and 15?

Reply:
15 is what we use in production, but also noticed that for some operations on my local, stuff was being bottlenecked and taking forever to backfill stuff one a single connection.  I added the acquire as I sometimes get a weird timeout bug when starting out, not sure why, but this seems to fix it after reading some stuff online.  I think we need to update to latest sequelize to fix some of these issues.